themes/inpycon2019/templates/article.html

```diff
@@ -8,10 +8,12 @@ <h1 class="title"> {{article.title}}</h1>
 
     <div class='meta'>
       <time datetime="{{article.date}}" itemprop="datePublished">
-	<small>Published: {{ article.date.strftime('%B %d, %Y') }}</small>
+	    <small>
+          Published: {{ article.date.strftime('%B %d, %Y') }}  |  Author: {{ article.author | default("PyCon India Content Team", true) }}
```

Comment:
Can you try linking Author's profile link as well ? See if you can provide the link in Author item itself.

Reply:
What is an Author profile link?

Reply:
Any link to author's bio

Reply:
@ananyo2012 The max that can be done is getting each author their author pages if you find a way please let me know. I am closing this PR and merging this change into #266